server.js

```diff
@@ -11,37 +11,68 @@ let mongoStream;
 // returns the status of all change streams currently running
 app.get('/', (request, response) => {
   const changeStreams = Object.keys(mongoStream.changeStreams);
-  const responseBody = {};
+  const responseBody = {total: changeStreams.length};
   for (let i = 0; i < changeStreams.length; i++) {
-    if (mongoStream.changeStreams[changeStreams[i]]) {
-      responseBody[changeStreams[i]] = 'Listening'
-    }
-    else {
-      responseBody[changeStreams[i]] = 'Not Listening'
-    }
+    const changeStream = mongoStream.changeStreams[changeStreams[i]];
+    if (changeStream)
+      responseBody[changeStream.collection] = 'Listening';
+    else
+      responseBody[changeStreams[i]] = 'Not Listening';
   }
 
   response.send(responseBody);
 });
 
-// triggers a manual collection dump for the specified collection
-app.get('/dump/:collection', (request, response) => {
-  console.log(`dumping collection ${request.params.collection} to ES`);
-  mongoStream.collectionDump(request.params.collection);
-  response.send(`dumping collection ${request.params.collection} to ES`);
+// triggers an add for the specified collections
+// @param collections: comma-separated string of collections to operate on
+// @param filter(optional): if exclusive, exclude the defined collections, else include them
+app.get('/add/:collections/:filter?', (request, response) => {
```

Comment:
Should be a POST or PUT.

Reply:
Could go either way here, but definitely not `GET`. "When in doubt, use `POST`" is a decent rule-of-thumb.